src/components/Alphabet/AlphabetPresentationSelected.js

```diff
@@ -110,7 +110,7 @@ function AlphabetPresentationSelected({
                 <div className="flex flex-col p-2 text-center">
                   {word?.translations?.length > 0 &&
                     word?.translations.map((translation) => (
-                      <p key={translation.id}>{translation?.text}</p>
+                      <p key={translation?.text}>{translation?.text}</p>
```

Comment:
did we break this recently or was this just a gap from before?

Reply:
I think it was a gap from before. Though, ideally we would have ids on all list items where possible.